packages/next-swc/README.md

```diff
@@ -36,16 +36,15 @@ pnpm build-wasm
 Due to platform differences napi bindings selectively enables supported features.
 See below tables for the currently enabled features.
 
-| arch\platform | Linux(gnu) | Linux(musl) | Darwin    | Win32     |
+| arch\platform | Linux(gnu) | Linux(musl) | Darwin    | Windows   |
 | ------------- | ---------- | ----------- | --------- | --------- |
-| ia32          |            |             |           | a,b,d,e   |
```

Comment:
We dropped support for 32-bit x86 Windows recently.

(also ["win32" is not technically correct here](https://en.wikipedia.org/wiki/Windows_API#Major_versions)?)